code/game/objects/items/storage/dakis.dm

```diff
@@ -22,7 +22,7 @@
 /obj/item/storage/daki/attack_self(mob/living/user)
 	var/body_choice
 	var/custom_name
-
+	user.setClickCooldown(1 SECONDS)
```

Comment:
The hotkey for using items in your active hand bypasses click delay. This here won't affect the majority of dakispam instances.